README.md

```
@@ -104,80 +236,130 @@
+- [EVE Online](https://www.eveonline.com/) and CCP Games
+- The Elixir/Phoenix community
+
+## Support
+
+- **Issues**: [GitHub Issues](https://github.com/wanderer-industries/wanderer-kills/issues)
+- **Discussions**: [GitHub Discussions](https://github.com/wanderer-industries/wanderer-kills/discussions)
+- **Email**: wanderer-kills@proton.me
+
+---
+
+Built with ❤️ by [Wanderer Industries](https://github.com/wanderer-industries)
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)